config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_dili.yml

```diff
@@ -806,135 +874,151 @@ patient:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id199
+    enum:
     - '0'
     - '1'
     type: string
     search_term: liver biopsy
+    limit: 2
   LiverRelatedDeath:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id201
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: death
+    search_term: cause of death
+    limit: 2
   LiverTransplant:
     categories:
     - biolink:ClinicalIntervention
-    enum: &id202
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: liver transplant
+    search_term: liver transplant planned
+    limit: 1
   MalignancyDx:
     categories:
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id204
+    enum:
     - '0'
     - '1'
     type: string
     search_term: malignancy
+    limit: 1
   NAFLD_Dx:
     categories:
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id213
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: NAFLD, non alcoholic fatty liver disease
+    search_term: non alcoholic fatty liver disease
+    limit: 2
   Nausea:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id215
+    enum:
     - '0'
     - '1'
     type: string
     search_term: nausea
+    limit: 3
   NeurologicalDiseaseDx:
     categories:
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id217
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: neurological disease
+    search_term: neurologic disease
+    limit: 1
   Neutropenia:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id218
+    enum:
     - '0'
     - '1'
     type: string
     search_term: neutropenia
+    limit: 3
   PCA_Race:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id228
+    enum:
     - White
     - Black
     - East Asian
     - Hispanic
     - Other/Multiracial
     type: string
-    search_term: race
+    search_term: racial group
+    limit: 4
   PatternLiverInjury_Onset:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id231
+    enum:
     - Cholestatic
     - Hepatocellular
     - Mixed
     type: string
-    search_term: cholestatic liver disease, hepatocellular liver disease
+    search_term: cholestatic liver
+    limit: 2
+    search_term: hepatocellular liver
+    limit: 2

```

Comment:
Duplicate search terms

Reply:
Fixed (hope this is the last one).

Reply:
@maximusunc : Are you ready to approve this PR? Or, are there additional edits that need to be made?

Reply:
Can I run a script that will make all features with search term have the new `name_lookup` structure? That way everything is consistent and easier for you to add new ones down the line.

Reply:
Most definitely. Thanks!